src/components/popup/details/vendors/vendors.jsx

```diff
@@ -41,11 +41,6 @@ export default class Vendors extends Component {
 	};
 
 	handleMoreChoices = () => {
-		const { consentCreated, initialVendorsRejection } = this.props;
```

Comment:
These are very important lines of code. If a user without cookies clicks a direct link to the vendor's view, consent should be unchecked. 
![image](https://user-images.githubusercontent.com/38684131/142200754-b4e41f82-6470-4809-9689-ddcff21f3173.png)